docs/source/concepts/processes.rst

```diff
@@ -108,39 +108,36 @@ The builder will help you in defining the inputs that the ``TemplatereplacerCalc
 
 Defining inputs
 ---------------
-For starters, if you are in an interactive shell and you simply evaluate the ``builder``, it will show you the inputs that it can take::
+For starters, if you are in an interactive shell and you simply evaluate the ``builder``, it will show you the current values that have been set for the inputs::
 
     builder
     {
-        'code': None,
-        'description': None,
-        'parameters': None,
-        'label': None,
-        'options': None,
-        'template': None,
-        'store_provenance': True [default]
+        'options': {},
     }
 
-Each key in the dictionary is an input of the ``TemplatereplacerCalculation`` class and the value is the current value that is set.
-If the calculation class defined a default value for an input (e.g. the ``store_provenance`` input in this example), it will be filled in and marked with ``[default]``.
-Another way to reveal the available inputs is through tab completion.
-In an interactive python shell, simply typing ``builder.`` and hitting the tab key, will show an autocomplete list of all available inputs.
+By evaluating the builder, it will display all the inputs that have been explicitly defined.
+In this example we have created a new instance of the builder and therefore, since no inputs have been set yet, the displayed dictionary is almost empty.
+Except for explicitly defined inputs, when evaluating the builder, the output will also show any nested namespaces that it has, such as the ``options`` namespace in this example.
+
+To find out which inputs the builder exposes, you can simply use tab completion.
+In an interactive python shell, by simply typing ``builder.`` and hitting the tab key, an autocomplete list of all available inputs will be shown.
```

Comment:
Can we add here that also options has tab completion for ``builder.options.``, and mention what they are briefly (e.g. that you need to define at lest the wall clock time and the resources, and link to the resources format in the Scheduler docs?